ext/imap/php_imap.c

```diff
@@ -539,13 +539,13 @@ static const zend_function_entry imap_functions[] = {
 #endif
 
 	PHP_FE(imap_mail,								arginfo_imap_mail)
+	PHP_FE(imap_getsubscribed,						arginfo_imap_getsubscribed)
+	PHP_FE(imap_getmailboxes						arginfo_imap_getmailboxes)
```

Comment:
Missing comma

Reply:
BAH! 

Do the travis builds (that passed)? not build imap ext? I guess it's probably time I had a local env to build and avoid these sort of errors.